packages/rancher-windows-exporter/charts/scripts/check-wins-version.ps1

```diff
@@ -0,0 +1,20 @@
+$ErrorActionPreference = 'Stop'
+
+$winsPath = "C:\Windows\wins.exe"
+$minWinsVersion = [System.Version]"0.1.0"
+
+function Get-Wins-Version
+{
+    $winsAppInfo = Invoke-Expression "& $winsPath cli app info | ConvertFrom-Json"
+    return [System.Version]($winsAppInfo.Server.Version.substring(1))
+}
+
+# Wait till the wins version installed is at least v0.1.0
+$winsVersion = Get-Wins-Version
+while ($winsVersion -lt $minWinsVersion) {
+    Write-Host $('wins on host must be at least v{0}, found v{1}. Checking again in 10 seconds...' -f $minWinsVersion, $winsVersion)
+    Start-Sleep -s 10
```

Comment:
Think we could shorten this?

Reply:
Yeah probably. I wanted to do:

```powershell
while (Get-Wins-Version -lt $minWinsVersion) {
    Write-Host $('wins on host must be at least v{0}, found v{1}. Checking again in 10 seconds...' -f $minWinsVersion, $winsVersion)
    Start-Sleep -s 10
```

But that seemed to cause issues with the `-lt` command.

Any suggestions?

Reply:
I actually meant the Sleep timer :) Could we do 5 seconds instead? Asking for UX reasons